### ISSUE TITLE
OneDrive Integration not Single Tenant compatible

### ISSUE THREAD

**Issue by AaronBoll-Kimoknow** [jgraph/drawio#3203]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
Registering the app as Single Tenant app in a Azure AD will result in an error (pop-up window stating access denied) every time a user tries to log in (Azure AD error code `50194`).
The reason for this seems to be that microsoft only allows multi tenant apps to use the https://login.microsoftonline.com/common endpoint, single tenant app are required to use a tenant specific endpoint (i.e. https://login.microsoftonline.com/${Tenant_UUID}).
The ability to use single tenant configurations would be greatly appreciated in a self hosted scenario.

Error Message in Azure AD Logs:
```
Application '{appId}'({appName}) is not configured as a multi-tenant application. Usage of the /common endpoint is not supported for such applications created after '{time}'. Use a tenant-specific endpoint or configure the application to be multi-tenant.
```

Login started working after setting the Azure AD App to multi tenant.

**To Reproduce**
1. Register a drawio server as an app in Azure AD
2. In Azure AD App settings set "Supported account types" to "Accounts in this organizational directory only"
3. Try logging into OneDrive --> Error
4. Reconfigure Azure AD App to multi tenant ("Accounts in any organizational directory")
5. Wait for AD to sync (takes a few minutes)
6. Try logging in --> works

**Expected behavior**
Login works.

**draw.io version (In the Help->About menu of the draw.io editor):**
20.6.0

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser MS Edge, Firefox

**Comment by m-mohamedin** [comment]
Thanks for the report. Support for single tenant will be added to the next release using `window.DRAWIO_MSGRAPH_TENANT_ID`

**Comment by davidjgraph** [comment]
Internal commit https://github.com/jgraph/drawio-dev/commit/dc03290718514b4fe470c45836ef1d332930c63f

**Comment by davidjgraph** [comment]
Released in 20.6.1
